frontend/packages/vkt/src/components/clerkExamEvent/overview/ClerkExamEventDetails.tsx

```diff
@@ -279,25 +281,35 @@ export const ClerkExamEventDetails = () => {
           onEdit={onEdit}
           onSave={onSave}
           isViewMode={isViewMode}
-          hasRequiredDetails={hasRequiredDetails}
+          isValidExamEvent={ExamCreateEventUtils.isValidExamEvent(
+            examEventDetails
+          )}
         />
       </div>
-      <ClerkExamEventDetailsFields
-        examEvent={examEventDetails}
-        onComboBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
-          handleComboBoxChange(field)
-        }
-        onDateChange={(
-          field: keyof Pick<
-            ClerkExamEventBasicInformation,
-            'date' | 'registrationCloses'
-          >
-        ) => handleDateChange(field)}
-        onCheckBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
-          handleCheckBoxChange(field)
-        }
-        editDisabled={isViewMode}
-      />
+      <div className="clerk-homepage__exam-events clerk-homepage-create-exam-events">
+        <ClerkExamEventDetailsFields
+          examEvent={examEventDetails}
+          onComboBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
+            handleComboBoxChange(field)
+          }
+          onDateChange={(
+            field: keyof Pick<
+              ClerkExamEventBasicInformation,
+              'date' | 'registrationCloses'
+            >
+          ) => handleDateChange(field)}
+          onCheckBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
+            handleCheckBoxChange(field)
+          }
```

Comment:
Eiks nää voi lyhentää näin 🤔 En siis kokeillu mutta näyttäis äkkiseltään

```suggestion
          onComboBoxChange={handleComboBoxChange}
          onDateChange={handleDateChange}
          onCheckBoxChange={handleCheckBoxChange}
```